src/studio/src/designer/backend/Controllers/ReleasesController.cs

```diff
@@ -45,18 +45,7 @@ public ReleasesController(IReleaseService releaseService, IPipelineService pipel
         [HttpGet]
         [ApiConventionMethod(typeof(DefaultApiConventions), nameof(DefaultApiConventions.Get))]
         public async Task<SearchResults<ReleaseEntity>> Get([FromQuery] DocumentQueryModel query)
-        {
```

Comment:
removing a recent bug fix here.
releases are solely linked to applications, so no need for the name change here.

Reply:
should revert the file